cli/module_reload.go

```diff
@@ -174,7 +174,9 @@ func mutateModuleConfig(c *cli.Context, modules []ModuleMap, manifest moduleMani
 				warningf(c.App.Writer, "you're replacing a registry module. we're converting it to a local module")
```

Comment:
```suggestion
				warningf(c.App.Writer, "you're replacing a registry module. we're converting it to a local module. To revert this change, use your machine's history page on app.viam.com")
```

[opt] In the meantime while we are doing damage to the config?

Reply:
(manually applied for line-length lint)